test/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+additional_flags = [
```

Comment:
Thought we might want to do things like this for other backends too at some point (aml/azure/local?).